packages/framework/undo-redo/src/mapHandler.ts

```diff
@@ -15,16 +15,30 @@ export class SharedMapUndoRedoHandler {
 
     public attachMap(map: ISharedMap) {
         map.on("valueChanged", this.mapDeltaHandler);
+        map.on("rollback", this.mapRollbackHandler);
```

Comment:
Is there a reason that clear is not supported here?

Reply:
i believe clear just fires a value change event for all keys. like i call out above the goal is to reduce the complexity for consumers. they just watch value changed and know they will see all changes to all keys regardless of source.